.github/workflows/ci.yml

```
@@ -35,7 +35,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['3.0', '3.1', '3.2']
+        ruby: ['3.1', '3.2', '3.3', '3.4']
```

For consistency with other version lists here 😄 

```suggestion
        ruby: ['3.4', '3.3', '3.2', '3.1']
```